### DISCUSSION TITLE
LGTM distribution configuration

### DISCUSSION THREAD

**Discussion by LorenDB** [community/community#21425] [Other Feature Feedback, Questions, & Ideas]
<p>I have an app written in C++ using Qt that I’d like to scan with LGTM. Unfortunately, while I’m requiring Qt 5.15 or higher, whatever distribution used under the hood doesn’t provide a new enough version of Qt. Therefore, I’d like to be able to either update the Qt packages or use an upgraded distribution altogether.</p>
<p>Does anybody have any information on how to do this?</p>

**Comment by LorenDB** [comment]
<p>Also, my IDE auto-generated a CMakeLists.txt that has CMake version 3.14 required. This is not a particular requirement, but the LGTM error message shows me that LGTM has only version 3.13.4, which matches the Debian Buster version.</p>

**Comment by ossanna16** [comment, marked as answer]
<p>Hi thank you for your post! This is not a GitHub related post so you may find more help in more general forums like Stackoverflow.</p>
